pages/docs/widgets/variants/button.en.mdx

```diff
@@ -18,19 +19,31 @@ The widget has several configuration parameters, which you can read about below.
 Add a piece of HTML to your web page with class `beequip-button`, in a location where you want the button to render.
 
 ```html
-<a href="https://lease.beequip.nl" class="beequip-button">Lease equipment bij Beequip</a>
+<a href="https://lease.beequip.nl" class="beequip-button"
+    >Lease equipment bij Beequip</a
+>
 ```
 
 Next, add the [required parameters](#required-parameters) with data-attributes, as described below.
 
 ```html
-<a href="https://lease.beequip.nl" class="beequip-button" data-token="Xhpx7QT6xRF9ZHyABrxmm12Q">Lease equipment bij Beequip</a>
+<a
```

Comment:
Could you revert the formatting changes, and configure your editor to ignore Markdown files?